src/api/models.ts

```diff
@@ -12,14 +12,35 @@ export const registerRequest = {
 export const loginRequest = {
   body: createSchema({
     password: Joi.string().required(),
-    login: Joi.string().required()
+    login: Joi.string().required(),
+    code: Joi.number()
```

Comment:
Lepsza czytelnosc jak sie napisze tak:
```suggestion
    code: Joi.number().optional()
```